### PR TITLE
Added missing asset for magic monarch & added eth wire fleece

### PR DESCRIPTION
**Pull request by bjoerm**

### CODE REVIEW

File: params.ini [hunk 1]
```diff
@@ -248,6 +248,7 @@ eth_armor_light_plate=1
 eth_armor_mage_plate=1
 eth_armor_sacred_armor=1
 eth_armor_shadow_plate=1
+eth_armor_wire_fleece=1
```
Comment (aeon0) [inline params.ini:251]: Either default this to 0 or add it also for 1280x720 please

**Review by aeon0 — CHANGES_REQUESTED** [review]
Thx for the updates. Just a little change request please.

**Comment by bjoerm** [comment]
I changed my PR based on your review input and created another commit for this PR (https://github.com/aeon0/botty/pull/70/commits/70602353de36e06c2fa9895ba8f9d15a3b3e39eb). Is this the correct way or do you need anything else from me?

**Review by aeon0 — APPROVED** [review]

**Comment by aeon0** [comment]
> I changed my PR based on your review input and created another commit for this PR ([7060235](https://github.com/aeon0/botty/commit/70602353de36e06c2fa9895ba8f9d15a3b3e39eb)). Is this the correct way or do you need anything else from me?

Nope, that's how I wanted it : ) thx

**Pull request merged**